src/index.html

```diff
@@ -6,7 +6,334 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
```

Comment:
You have two logo classes. Also, don't need to create a new bem block just for the logo. It's small element
```suggestion
    <header class="header">
      <a href="#home" class="lheader__logo-link">
        <img
          src="images/logo.png"
          alt="logo"
          class="header__logo"
```